domain/cmd/migrate.go

```
@@ -0,0 +1,128 @@
+	},
+}
+
+var downCmd = &cobra.Command{
+	Use:   "down",
+	Short: "Reset all migrations",
+	Long:  "Revert all migrations",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		fmt.Println("WARNING: This will reset ALL migrations. All data will be lost.")
+		fmt.Print("Are you sure you want to continue? (yes/no): ")
+		var response string
+		fmt.Scanln(&response)
```

golangci-lint (1.64.8)

Error return value of `fmt.Scanln` is not checked

(errcheck)